### PR TITLE
Keysmith - initial inclusion

### PR DESCRIPTION
**Pull request by Justinzobel**
**Summary**

Initial inclusion

**Test Plan**

Copied my TOTP keys from another device and tested, all OK

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/k/keysmith/package.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    - https://download.kde.org/stable/release-service/23.08.4/src/keysmith-23.08.4.tar.xz : 76100bf9a038d8bbfdcfe434c02c8362f654c70051b2c1e13055ff7b2031d585
+homepage   : https://apps.kde.org/keysmith
+license    : GPL-3.0-or-later
+component  : desktop.kde
+summary    : OTP client for Plasma Mobile and Desktop
+description: |
+    Keysmith is an application to generate two-factor authentication (2FA) tokens when logging in to your (online) accounts. Currently it supports both HOTP and TOTP tokens.
+builddeps  :
+    - pkgconfig(libsodium)
+    - pkgconfig(Qt5Core)
```
Comment (ReillyBrogan) [inline packages/k/keysmith/package.yml:14]: Capitalized pkgconfig deps are ordered above lowercase ones, so the three `Qt5*` deps should be above the `libsodium` one.

File: packages/k/keysmith/package.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    - pkgconfig(Qt5QuickControls2)
+    - pkgconfig(Qt5Svg)
+    - extra-cmake-modules
+    - kirigami2-devel
+    - ki18n-devel
+    - kdbusaddons-devel
```
Comment (ReillyBrogan) [inline packages/k/keysmith/package.yml:20]: These three need to be in alphabetical order.

**Review by ReillyBrogan — CHANGES_REQUESTED** [review]
I also see that the plasma-wallpapers commit is still in the "not in main" part of this branch (see the "Commits" tab on the PR). Given that plasma-wallpapers has already been merged into main I think this likely just needs a rebase. Try the following and see if it fixes it:
```
git fetch origin
git rebase origin/main
git push --force
```

**Comment by ReillyBrogan** [comment]
I assume that the reason the plasma-wallpapers commit is included in this branch is because you created your branch from the plasma-wallpapers one. May I introduce you to the following command:
```
git switch -c new-branch-name origin/main
```
Which will create a new branch named `new-branch-name` and then switch to it. `origin/main` is the starting point for the new branch instead of the current HEAD. Bonus points if you run `git fetch origin` first so that the branch is fully up-to-date with Github.

**Comment by Justinzobel** [comment]
Fixed in the other PR.

**Pull request closed without merging**